### ISSUE TITLE
File system monitoring

### ISSUE THREAD

**Issue by rossknudsen** [kavod-io/vscode-jest-test-adapter#13]
Monitor the following files:

- [x] ~Added config files should cause additional tests to be found.  This probably depends on #9 being implemented first.  In the meantime, we should invalidate all tests.~
- [x] ~Removed config files should cause tests to be removed.  This probably depends on #9 being implemented first.  In the meantime, we should invalidate all tests.~
- [x] ~Modified config files should cause tests to be updated.  This probably depends on #9 being implemented first.  In the meantime, we should invalidate all tests.~
- [x] Added test files should add tests/suites
- [x] Removed test files should remove tests/suites
- [x] Modified test files should update the tests/suites and invalidate all existing tests in that suite.
- [x] Added app files should not invalidate anything.  We'll assume that an added file does not contain any code which changes the outcomes of tests.
- [x] Removed app files should invalidate *all* tests/suites.  This is until we can use Jest to tell us which tests are affected.
- [x] Updated app files should invalidate *all* tests/suites.  This is until we can use Jest to tell us which tests are affected.

Monitoring of config files will be moved to a separate issue.
